djangocms_navigation/cms_config.py

```diff
@@ -83,6 +86,8 @@ def on_menu_content_archive(version):
 
 
 class NavigationCMSAppConfig(CMSAppConfig):
+    # NOTE what to do in sitemap. Very similar. It will register the correct model
+    # with the config. But navigation models may be different. Settings for this?
```

Comment:
@Aiky30 How would you like to solve this, any ideas? The only difference is `navigation_models`.

Reply:
I got this.